VERSION

```diff
@@ -1 +1 @@
-1.2.0
+1.3.0
```

Comment:
Please don't bump this until we do a release.

Reply:
Reverted version number